src/plugins/obj/obj_backend.cpp

```
@@ -26,6 +26,24 @@
 // -----------------------------------------------------------------------------
 // Obj Engine Implementation
 // -----------------------------------------------------------------------------
+//
+// Engine selection:
+//   * Default (no params)         -> DefaultObjEngineImpl
+//   * crtMinLimit > 0             -> S3CrtObjEngineImpl
+//   * accelerated=true, type=...  -> registry-provided vendor engine (LEGACY)
+//
+// The idiomatic GPU-direct path is the S3-over-RDMA capability built into the
+// standard S3 client (see s3/rdma.h): it follows the published `x-amz-rdma-*`
```

Cppcheck (2.20.0)

[style] The function 'waitUntilIdle' is never used.

(unusedFunction)